index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quacker</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css">
+</head>
+<body>
+    <div class="container">
+        <nav class="left">
+            <a class="logo" href="#"><img src="https://www.pngpix.com/wp-content/uploads/2016/10/PNGPIX-COM-Duck-PNG-Transparent-Image-1.png" alt=""></a>
+            <a class="home" href="#">
+                <i class="bi bi-house"></i><span>Home</span>
+            </a>
+            <a class="explore" href="#">
+                <i class="bi bi-hash"></i><span>Explore</span>
+            </a>
+            <a class="notifications" href="#">
+                <i class="bi bi-bell"></i><span>Notifications</span>
+            </a>
+            <a class="messages" href="#">
+                <i class="bi bi-envelope"></i><span>Messages</span>
+            </a>
+            <a class="bookmarks" href="#">
+                <i class="bi bi-bookmarks"></i><span>Bookmarks</span>
+            </a>
+            <a class="profile" href="#">
+                <i class="bi bi-person"></i><span>Profile</span>
+            </a>
+            <a class="more" href="#">
+                <i class="bi bi-three-dots"></i><span>More</span>
+            </a>
+            <a class="button QUACK" href="#"><span>QUACK</span></a>
```

Comment:
This might need some align work on higher vertical resolutions.
![image](https://user-images.githubusercontent.com/113195009/206678611-c5ac5dab-5212-4735-bf78-bae12ea2a18e.png)